### PR TITLE
feat(preset-mini): Add support for peer and group variants on aria-* variants to Preset-Mini

### PR DESCRIPTION
**Pull request by Johnkat-Mj**
close: https://github.com/unocss/unocss/issues/3691

This pull request enable the use of peer and group selectors in conjunction with ara-* variants ( #3691 )
This will provide the ability to achieve styles like:

- peer-aria-expanded:bg-red: Applies a red background to an element when its peer element has the aria-expanded attribute set to true.

- group-aria-expanded-p-4: Adds a padding of 4 units to elements within a group when a group member has the aria-expanded attribute set to true.

Example
```
<button data-accordion-trigger aria-label="toggle button"
    aria-expanded="true"
    class="... group" ...>
    Can it be animated?
    <span
        class=".... group-aria-expanded-rotate-180"></span>
</button>
```
In this example, the `rotate-180`  will be applied to the span element when the group member (button) has the aria-expanded attribute set to true.

### CODE REVIEW

**Comment by zyyv** [comment]
@Johnkat-Mj Can you add some test cases?

**Comment by Johnkat-Mj** [comment]
@zyyv Can you guide me through this?
Should I just add a usage example in EXAMPLES or should I write a test for this?

File: docs/presets/community.md [hunk 1]
```diff
@@ -27,6 +27,7 @@
 - [unocss-preset-animations](https://unocss-preset-animations.aelita.me/) - Preset for flexible and composable animations by [@Aelita](https://github.com/xsjcTony).
 - [unocss-preset-shadcn](https://github.com/hyoban/unocss-preset-shadcn#readme) - Use [shadcn/ui](https://ui.shadcn.com) with UnoCSS by [@zouhang](https://github.com/zouhangwithsweet) and [@hyoban](https://github.com/hyoban).
 - [@yeungkc/unocss-preset-safe-area](https://github.com/YeungKC/unocss-preset-safe-area) - Use safe area with Unocss by [YeungKC](https://github.com/YeungKC)
+- [@unifyui/unify-preset](https://github.com/unify-ui-dev/unify-preset/blob/main/README.md) - Atomic Theming UI Libray Powered By UnoCSS, like DaisyUI but customizable. By [@johnkat-mj](https://github.com/Johnkat-Mj)
```
Comment (antfu) [inline docs/presets/community.md:30]: Can you do this with a separate PR? One PR should only do one thing if possible

**Comment by zyyv** [comment]
> @zyyv Can you guide me through this? Should I just add a usage example in EXAMPLES or should I write a test for this?

Like this https://github.com/unocss/unocss/blob/5ae78dfb3e4a9c6d5de326de6a9ac6de9b5e87c6/test/assets/preset-mini-targets.ts#L1176-L1177
Add your test cases, update the snapshot and check it right?

**Comment by Johnkat-Mj** [comment]
Hey @zyyv 
The test fails when I add `group-aria-[expanded=true]:bg-green-400` in `test/assets/preset-mini-targets.ts`, but I can't figure out the cause. After running `nr build`, I added an example in `examples/nuxt3`, and it worked fine as expected without any issues. Even the 'output' after running `pnpm build` showed no problems. However, I'm still unable to understand why the test is failing, the logic is the same as that of tagged data attributes

**Comment by Simon-He95** [comment]
close: #3691

**Review by antfu — APPROVED** [review]
